streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBWindowStore.java

```diff
@@ -47,13 +54,12 @@
 import java.util.SimpleTimeZone;
 import java.util.concurrent.ConcurrentHashMap;
 
-public class RocksDBWindowStore<K, V> implements WindowStore<K, V> {
+public class RocksDBWindowStore<K, V> implements WindowStore<K, V>, CanSendOldValues {
```

Comment:
For the window store the caching is done at the `Segment` level. The window store uses a modified key, i.e., `byte[] binaryKey = WindowStoreUtils.toBinaryKey(key, timestamp, seqnum, serdes);`, the caching layer doesn't know about this and it really shouldn't.